test/unit/metrics/test_fairness_metrics.py

```diff
@@ -79,23 +83,26 @@ def test_demographic_parity_ratio_weighted(agg_method):
     assert actual == gm.ratio(method=agg_method)
 
 
-@pytest.mark.parametrize("agg_method", _aggregate_methods)
-def test_equalized_odds_difference(agg_method):
+@pytest.mark.parametrize("agg_method, agg", _agg_combinations)
+def test_equalized_odds_difference(agg_method, agg):
     actual = equalized_odds_difference(
-        y_t, y_p, sensitive_features=g_1, method=agg_method
+        y_t, y_p, sensitive_features=g_1, method=agg_method, agg=agg
     )
 
     metrics = {"tpr": true_positive_rate, "fpr": false_positive_rate}
     gm = MetricFrame(metrics=metrics, y_true=y_t, y_pred=y_p, sensitive_features=g_1)
 
     diffs = gm.difference(method=agg_method)
-    assert actual == diffs.max()
+    if agg == "worst_case":
+        assert actual == diffs.max()
+    else:
```

Comment:
Nitpick: for future maintainability it might be nicer to replace `else` by an explicit `if agg == "mean"`.

Reply:
I'm testing the options above, so if it's anything else we can't get here. Are you suggesting I don't check at the beginning but only here at the end? I guess I preferred the early return/raise pattern over doing all the calculations just to let users know that we won't in fact allow other inputs for agg.

Reply:
Fair enough. I always like it when tests are very explicit so i don't have to scroll up to see if it makes sense, and it's a bit less error prone in case we ever decide to add another `agg` option. But it's truly a nitpick :)